Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs

```diff
@@ -149,8 +152,7 @@ public ISiteBuilder CreateBuilder(ITracer tracer, ILogger logger, IDeploymentSet
         private ISiteBuilder ResolveNonAspProject(string repositoryRoot, string projectPath, IDeploymentSettingsManager perDeploymentSettings)
         {
             string sourceProjectPath = projectPath ?? repositoryRoot;
-            // "FUNCTIONS_EXTENSION_VERSION" environment variable implies a functionApp
-            if (FunctionAppHelper.LooksLikeFunctionApp())
+            if (IsFunctionApp(sourceProjectPath))
```

Comment:
Why move this into an additional level of helper if all it does is call `FunctionAppHelper.LooksLikeFunctionApp`?